en_US/changes/breaking-changes-ee-5.8.md

```diff
@@ -1,5 +1,12 @@
 # Incompatible Changes in EMQX 5.8
 
+## e5.8.1
+
+- [#13792](https://github.com/emqx/emqx/pull/13792) The default expiration time for a banned item that is created without an `until` value is now `infinity` (previsouly capped at 1 year limit).
```

Comment:
both breaking changes should be added to ce as well

Reply:
done